docs/specviz2d/plugins.rst

```diff
@@ -94,9 +94,11 @@ The background step of the plugin allows for creating background and background-
 images via `specreduce.background <https://specreduce.readthedocs.io/en/latest/#module-specreduce.background>`_.
 
 Once you interact with any of the inputs in the background step or hover over that area
-of the plugin, the live visualization will change to show the center (dotted line) and edges
-(solid lines) of the background region(s).  Choose between creating the background
-around the trace defined in the Trace section, or around a "Manual" flat trace.
+of the plugin, the live visualization in the 2d spectrum viewer will change to show the center 
+(dotted line) and edges (solid lines) of the background region(s).  The 1D representation of the
+background will also be visualized in the 1D spectrum viewer (thin solid line).
+
+Choose between creating the background around the trace defined in the Trace section, or around a "Manual" flat trace.
```

Comment:
I found this sentence a little hard to parse. What about something like this? "Backgrounds can either be created around the trace defined in the earlier Trace section or around a new, flat trace by selecting a "Manual" Background type."

Reply:
slightly modified your suggestion to '... by selecting "Manual" in the Background Type dropdown'